components/text/less/Makefile

```diff
@@ -22,20 +22,21 @@
 #
 # Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
 # Copyright (c) 2019, Michal Nowak
+# Copyright (c) 2020, Nona Hansel
 #
 
 PREFERRED_BITS=		64
 
```

Comment:
Can you please also change to the new style:
Replace
PREFERRED_BITS=64
with
BUILD_BITS=64

Reply:
Changed it.